app/src/main/java/pl/gittobefit/running_training/viewmodel/TrainingViewModel.java

```diff
@@ -0,0 +1,112 @@
+package pl.gittobefit.running_training.viewmodel;
+
+import android.content.Context;
+
+import androidx.lifecycle.ViewModel;
+
+import java.util.ArrayList;
+
+import pl.gittobefit.database.entity.training.Exercise;
+import pl.gittobefit.database.entity.training.relation.TrainingWithForm;
+import pl.gittobefit.database.pojo.ExerciseExecutionPOJODB;
+import pl.gittobefit.database.repository.TrainingRepository;
+
+public class TrainingViewModel extends ViewModel {
+
+    private TrainingWithForm trainingWithForm;
+    private TrainingRepository trainingRepository;
+    private ArrayList<Exercise> listExercises;
+    private ArrayList<ExerciseExecutionPOJODB> planList;
+    private int indexExercise;
+    public int numberOfSeries;
```

Comment:
to mozesz spowrotem dac na private

Reply:
Ja bym powiedział, że jeżeli nic nie wymaga tego, żeby to było public to nawet trzeba